pkgs/applications/video/gnome-mpv/no-appdata-validate.patch

```diff
@@ -0,0 +1,14 @@
+--- a/data/meson.build
++++ b/data/meson.build
+@@ -41,11 +41,6 @@ configure_file(
+   install_dir: join_paths(get_option('datadir'), 'dbus-1', 'services')
+ )
+ 
+-appstream_util = find_program('appstream-util', required: false)
+-if appstream_util.found()
+-  test('Validate appstream file', appstream_util, args: ['validate', appdata])
```

Comment:
Adding `--nonet` flag is the proper fix:

https://gitlab.gnome.org/GNOME/buoh/blob/792b6e08d9101919202ccf9d4b365d5e04e2b050/data/meson.build#L80

Can be even be upstreamed.

Reply:
That's much cleaner, thanks. Updated.